src/GUI/UI/Houses/Tavern.h

```diff
@@ -3,4 +3,21 @@
 #include "GUI/UI/UIHouses.h"
 #include "GUI/UI/UIHouseEnums.h"
 
-void TavernDialog();
+class GUIWindow_Tavern : public GUIWindow_House {
```

Comment:
probably can rename to class Tavern at this point
GUIWindow_ prefix is purely historical from the times where there were no classes at all, only functions and global variables

Reply:
Or TavernWindow / TavernDialog, yes.

Reply:
OK to do in a separate PR